errors/Readme.md

```diff
@@ -240,7 +240,7 @@ SDK acts as the interface between the binaries and the tools. This section cover
 
 #### P1003: Database does not exist
 
-- **Description**: MySQL, Postgres: Database `${database_name}` does not exist on the database server at `${database_host}`. <br /> <br /> SQLite: Database `${database_file_name}` does not exist on the database server at `${database_file_path}`
+- **Description**: Database `${database_name}` does not exist on the database server at `${database_host}`.
```

Comment:
This change removes the information of the file path for SQLite. 

Probably @divyenduz already meant this as alternative descriptions to be used and adapted on a case by case basis, but this should probably be made explicit:

```suggestion
MySQL, Postgres:
- **Description**: Database `${database_name}` does not exist on the database server at `${database_host}`.

SQLite:
- **Description**: Database `${database_file_name}` does not exist at `${database_file_path}`.
```

Reply:
Update> Oh, did you actually mean P1003 in your description instead of P1001?

Reply:
Sorry yes, P1003. The point of the change was to avoid having multiple messages mapped to one error code, with different variable names. We can definitely implement this but this seemed like unnecessary complexity, since the message will be the same. I replaced `database_host` with `database_location` in the Meta definition, but forgot to update the variable in the template string. `database_location` is the file path on sqlite and the host on mysql/postgres.

Reply:
I'll remove that change from this PR so we can merge the trivial fixes and open an issue (will link it here as well).

Reply:
https://github.com/prisma/specs/issues/314